lib/secure_headers/headers/policy_management.rb

```diff
@@ -325,10 +329,10 @@ def validate_sandbox_expression!(directive, sandbox_token_expression)
         return if boolean?(sandbox_token_expression) && sandbox_token_expression == true
         ensure_array_of_strings!(directive, sandbox_token_expression)
         valid = sandbox_token_expression.compact.all? do |v|
-          v.is_a?(String) && v.start_with?("allow-")
+          v.is_a?(String) && v.start_with?("allow-") && v !~ ESCAPE_SEQUENCE
```

Comment:
It would probably be cleaner to move this check to `ensure_array_of_strings`.

But the belt-and-suspenders approach of also raising an error on output is also probably a good idea.